tools/test.py

```diff
@@ -237,8 +237,9 @@ def main():
             model.cuda(),
             device_ids=[torch.cuda.current_device()],
             broadcast_buffers=False)
-        outputs = multi_gpu_test(model, data_loader, args.tmpdir,
-                                 args.gpu_collect)
+        outputs = multi_gpu_test(
+            model, data_loader, args.tmpdir, args.gpu_collect
+            or cfg.evaluation.gpu_collect)
```

Comment:
We should handle the case when both args.gpu_collect and cfg.evaluation.gpu_collect is set. It might be better that args.gpu_collect has higher priority than cfg.evaluation.gpu_collect

Reply:
args.gpu_collect presently cannot be set to false. So if args.gpu_collect is set, it does have a higher priority. 

The case in question is when cfg.evaluation.gpu_collect is true. Users cannot disable gpu_collect through args.gpu_collect. Is this behaviour acceptable?

Reply:
args.gpu_collect presently cannot be set to false. So if args.gpu_collect is set, it does have a higher priority. 

The case in question is when cfg.evaluation.gpu_collect is true. Users cannot disable gpu_collect through args.gpu_collect. Is this behavior acceptable?

Reply:
Yes. I think this behavior is acceptable.